src/main/java/com/google/sps/placeGuide/repository/PlaceGuideRepository.java

```diff
@@ -23,5 +26,9 @@
   // (not available to be viewed by other users).
   public List<PlaceGuide> getCreatedPrivatePlaceGuides(String creatorId);
 
+  public List<PlaceGuide> getBookmarkedPlaceGuides(String userId);
+
+  public void removeBookmarkedPlaceGuide(long placeGuideId, String userId);
+
```

Comment:
The previous remark is valid for this function as well. 
You may leave getBookmarkedPlaceGuides here, but I think it would fit into the UserRepository too.

Reply:
done.